app/assets/stylesheets/_settings.scss

```diff
@@ -334,7 +334,7 @@ $input-prefix-color: $black;
 $input-prefix-background: $light-gray;
 $input-prefix-border: 1px solid $medium-gray;
 $input-prefix-padding: 1rem;
-$form-label-color: $black;
+$form-label-color: $white;
 $form-label-font-size: rem-calc(14);
 $form-label-font-weight: $global-weight-normal;
 $form-label-line-height: 1.8;
```

Comment:
Going with light text on a dark background, I see? There's nothing _technically_ wrong with this...